test/getUriParameters-test.coffee

```diff
@@ -189,6 +189,73 @@ describe('Transformation • Refract • getUriParameters' , ->
           }
         ]
       },
+      {
+        hrefVariables: {
+          "element": "hrefVariables",
+          "meta": {},
+          "attributes": {},
+          "content": [
+            {
+              "element": "member",
+              "meta": {},
+              "attributes": {},
+              "content": {
+                "key": {
+                  "element": "string",
+                  "meta": {},
+                  "attributes": {},
+                  "content": "filters"
+                },
+                "value": {
+                  "element": "array",
```

Comment:
**@XVincentX Could you please share which Swagger document produces this response please.

I'm not sure it makes sense to have an array of enums and this may be a bug in the Swagger adapter.

For example, here's an API Blueprint:

```apib
# Resource [/test{?a}]

+ Parameters
    + a (enum)
        + Members
            + 1
            + 2
            + 3
```

Which produces the following:

```json
{
  "element": "parseResult",
  "content": [
    {
      "element": "category",
      "meta": {
        "classes": [
          "api"
        ],
        "title": ""
      },
      "content": [
        {
          "element": "category",
          "meta": {
            "classes": [
              "resourceGroup"
            ],
            "title": ""
          },
          "content": [
            {
              "element": "resource",
              "meta": {
                "title": "Resource"
              },
              "attributes": {
                "href": "/test{?a}",
                "hrefVariables": {
                  "element": "hrefVariables",
                  "content": [
                    {
                      "element": "member",
                      "attributes": {
                        "typeAttributes": [
                          "required"
                        ]
                      },
                      "content": {
                        "key": {
                          "element": "string",
                          "content": "a"
                        },
                        "value": {
                          "element": "enum",
                          "content": [
                            {
                              "element": "string",
                              "content": "1"
                            },
                            {
                              "element": "string",
                              "content": "2"
                            },
                            {
                              "element": "string",
                              "content": "3"
                            }
                          ]
                        }
                      }
                    }
                  ]
                }
              },
              "content": []
            }
          ]
        }
      ]
    }
  ]
}
```**